appveyor.yml

```diff
@@ -4,7 +4,7 @@ build:
   verbosity: minimal
 
 build_script:
-  - gradlew build
+  - gradlew build test
```

Comment:
See above comment with task tree. Looks to me like `build` will already run `test`, was that not your experience?

Reply:
No, but I was having issues with getting clear output that the tests had run. I have local changes for build.gradle to be more verbose which might prevent some of the confusion.